.github/workflows/client-celo.yml

```diff
@@ -70,5 +93,16 @@ jobs:
           # VERSION= ? TODO: Configure version, sample: 1.7.6
           push: |
             ${{ github.ref == 'refs/heads/master'
-              && (github.event_name == 'push'
-              || github.event_name == 'workflow_dispatch') }}
\ No newline at end of file
+              && github.event_name != 'pull_request' }}
+
+  client-scan:
```

Comment:
Can we revert this change and leave the scan function in `gosec.yml`? I'm not sure if `client-celo.yml` is a good place for the job.

Reply:
Sorry, it was a mixup on my side when I was manually copying/pasting code from different branch. I intended to have `gosec` in `client-ethereum` (as covered by https://github.com/keep-network/keep-ecdsa/pull/751/files), but by mistake I pasted it to the wrong file (the code I wanted to use to start a new branch was depending on two PRs and I ended up doing checkout from one and manually applying some changes from another - I should have use cherry-pick instead, or don't include the change at all and later deal with it via resolving merge conflicts).
Anyway, I restored the original config - see 46b71aedb51496efa534022a9482d060ce8f6b2a.